### ISSUE TITLE
Cargo deny warns against the dependency input-event-codes GPLv2 license as copyleft

### ISSUE THREAD

**Issue by coderedart** [ostrosco/device_query#51]
`device_query` uses `input-event-codes` crate as dependency, and has MIT as the license. 

but `input-event-codes` has its license set to `"GPL-2.0-only"` which is AFAICT copy-left which would mean device_query (and crates that depend on it) would get infected with that license too as all of us use static linking against that crate (and its source). 

I was just setting up `deny.toml` for my project and this issue popped up.

**Comment by coderedart** [comment]
Link to the relevant crate's `Cargo.toml` https://gitlab.com/gm666q/input-event-codes/-/blob/master/Cargo.toml

**Comment by ostrosco** [comment]
Good catch. I agree we'll need to move away from this then.

**Comment by ostrosco** [comment]
I've just put up a PR that should address this issue: https://github.com/ostrosco/device_query/pull/52

Once I get some other eyes on it, we should be good to go and I'll cut a new release.

**Comment by ostrosco** [comment]
I've published version 1.1.0 which resolves this issue. All of our licenses are compatible now so we should be good to go. Thanks for noticing this!

```
➜  device_query git:(master) cargo license
Apache-2.0 OR MIT (9): core-foundation, core-foundation-sys, lazy_static, libc, macos-accessibility-client, pkg-config, winapi, winapi-i686-pc-windows-gnu, winapi-x86_64-pc-windows-gnu
MIT (4): device_query, readkey, readmouse, x11
```
